DocApprove.php

```diff
@@ -53,14 +53,14 @@
 		if($obj['status'] == "Confirmed" )
```

Comment:
Where's the matching closing bracket for  `if(isset($_GET['process']))`?
I am not able to find it.